asapdiscovery-docking/asapdiscovery/docking/docking.py

```diff
@@ -204,6 +239,44 @@ def run_docking_oe(
         pose_res = oedocking.OEPositResults()
         ret_code = poser.Dock(pose_res, dock_lig, num_poses)
 
+    if allow_low_posit_prob and (
```

Comment:
@apayne97 I have added some last ditch efforts to generate a pose that you can turn on.